### ISSUE TITLE
Improve the readme of the project

### ISSUE THREAD

**Issue by prateek3255** [devfolioco/scrroll-in#32]
**Document a detailed readme for this repository describing how to use this extension and how does it work internally.**

You can use the [chrome webstore description](https://chrome.google.com/webstore/detail/scrroll-in/cjgjbjogfodppempgdlppgefojbcmjom?hl=en&gl=IN) as a reference.

For the internal working part, you can describe it by pointing out specific functions and files explaining their roles.

**Comment by luanorlandi** [comment]
I will help you with this issue

btw, I installed the extension and find it pretty useful :)

**Comment by Kuljeet-123** [comment]
@prateek3255 Can I work on this issue??

**Comment by adinalini** [comment]
I would love to work on this issue

**Comment by KalagaSahitya** [comment]
@prateek3255  I would like to work on this Issue

**Comment by CadencePrestissimo** [comment]
I would like to work on this Issue

**Comment by kanika321** [comment]
Can I work on this issue. I have an interest in documentation and would love to contribute.

**Comment by GoGi2712** [comment]
I really want to work on this issue.

**Comment by prateek3255** [comment]
Yeah, sure go ahead and please open meaningful PRs, explaining how this extension works internally and how it is used. Feel free to add screenshots.

**Please don't open spam PRs fixing irrelevant grammar, or changing `Scrroll In -> Scroll In`, it's not a typo.**

**Comment by brandonbawe** [comment]
I'd love to work on the documentation too. Never documented a project before but I'd really like to try this one.
